### DISCUSSION TITLE
[Feature] Add option to disable web client.

### DISCUSSION THREAD

**Discussion by micash545** [immich-app/immich#6278] [Feature Request]
Just add ENV variable to disable web client serving in server.

**Comment by Vera2016** [comment]
if you are using nginx now, try follwing code.

`
#immich
        server {
                listen 2238 ssl; 
                server_name your_domain.com; 
                ssl_certificate .../fullchain.pem;
                ssl_certificate_key .../private.key;
                # set timeout
                proxy_read_timeout 600s;
                proxy_send_timeout 600s;
                send_timeout       600s;
                client_max_body_size 50000M;
                # Set headers
                proxy_set_header Host $host;
                location /api {
                        proxy_pass http://192.168.1.1:2283; // immich server
                }
                location /.well-known {
                        proxy_pass http://192.168.1.1:2283;  // immich server
                }
                location / {
                        proxy_pass http://127.0.0.1; 
                }
        }

`
